src/resources/VoipTicket.js

```diff
@@ -34,10 +34,13 @@ class VoipTicket extends Resource {
 
   /**
    * Generates a new voip ticket via the client
+   * @param {string} provisionKey A constant key on header to retrieve sinch secret
    * @returns {Promise} If successful, a hydrated instance of a fresh voip ticket
    */
-  fetch() {
-    return this.client.post(this.href)
+  fetch(provisionKey) {
+    return this.client.post(this.href, { headers: {
+      'provision_key': provisionKey}
+    })
```

Comment:
I think this exposes a bug in non-browser environment support by clobbering the required `Content-Type` and `Accept` headers that [are set by default in Client.js](https://github.com/syncromatics/syncromatics-track-api/blob/master/src/Client.js#L98).  Could you please update Client.js as well to take into account merging any `headers` object that's passed in with the default headers?